### PR TITLE
Update docs of charts, minikube, and userguide

### PR DESCRIPTION
**Pull request by qinpingli**

### CODE REVIEW

File: docs/userguide.md [hunk 1]
```diff
@@ -59,7 +59,7 @@ Please refer to [Kubefed Concepts](./concepts.md) first before you go through th
 
 ## Prerequisites
 
-The kubefed deployment requires kubernetes version >= 1.11. The following
+The kubefed deployment requires kubernetes version >= 1.13. The following
```
Comment (marun) [inline docs/userguide.md]: Should this be updated?  Fine to use a newer version, but we want to support older versions of kube if possible.

cc: @pmorie

File: charts/index.yaml [hunk 2]
```diff
@@ -1,28 +1,28 @@
 apiVersion: v1
 entries:
-  federation-v2:
+  kubefed:
```
Comment (marun) [inline charts/index.yaml]: I don't think any of the previous releases should be renamed. They should remain in the index as federation-v2.

File: charts/index.yaml [hunk 3]
```diff
@@ -1,28 +1,28 @@
...
   - apiVersion: v1
     created: 2019-05-06T15:30:47.795749649-07:00
     description: Kubernetes Federation V2 helm chart
     digest: 9b248eed6f021951653ee98cb255d2d3564748ac04c2484c941f619c39c5dd79
-    name: federation-v2
+    name: kubefed
     urls:
-    - https://sigs.k8s.io/kubefed/releases/download/v0.0.10/federation-v2-0.0.10.tgz
+    - https://github.com/kubernetes-sigs/federation-v2/releases/download/v0.0.10/federation-v2-0.0.10.tgz
```
Comment (marun) [inline charts/index.yaml]: Good catch.  Looks like the links for old releases aren't changed by the rename.

cc: @pmorie

File: charts/index.yaml [hunk 4]
```diff
@@ -1,28 +1,28 @@
 apiVersion: v1
 entries:
-  federation-v2:
+  kubefed:
   - apiVersion: v1
     created: 2019-05-06T15:30:47.795749649-07:00
     description: Kubernetes Federation V2 helm chart
     digest: 9b248eed6f021951653ee98cb255d2d3564748ac04c2484c941f619c39c5dd79
-    name: federation-v2
+    name: kubefed
```
Comment (marun) [inline charts/index.yaml]: Same as previous comment - leave as federration-v2.

File: charts/kubefed/README.md [hunk 5]
```diff
@@ -77,7 +77,7 @@ $ helm search kubefed
 Install the chart and specify the version to install with the
 `--version` argument. Replace `<x.x.x>` with your desired version.
 ```bash
-$ helm install fedv2-charts/kubefed --version=<x.x.x> --namespace kube-federation-system
+$ helm install kubefed-charts/kubefed --version=<x.x.x> --namespace kube-federation-system
```
Comment (marun) [inline charts/kubefed/README.md]: This should be `kubefed-charts/federation-v2` for now.

File: docs/userguide.md (hunk 1 above)
Comment (qinpingli) [reply docs/userguide.md]: Thanks, @marun , I saw there is some updating for these docs, kubernetes version is updated to  >= 1.13, I'll merge the new updating, and update this PR.

File: charts/index.yaml (hunk 2 above)
Comment (qinpingli) [reply charts/index.yaml]: Got it.

File: charts/kubefed/README.md (hunk 5 above)
Comment (qinpingli) [reply charts/kubefed/README.md]: Got it.

File: charts/kubefed/README.md [hunk 6]
```diff
@@ -96,6 +96,7 @@ Delete all kubefed CRDs:
 
 ```bash
 $ kubectl delete crd $(kubectl get crd | grep -E 'kubefed.k8s.io' | awk '{print $1}')
+$ kubectl delete crd $(kubectl get crd | grep -E 'clusterregistry.k8s.io' | awk '{print $1}')
```
Comment (marun) [inline charts/kubefed/README.md]: This line is no longer necessary, as the cluster registry is no longer installed by the chart.  Please revert.

Comment (qinpingli) [reply charts/kubefed/README.md]: Done.

**Pull request merged**